tests/e2e-automatic-rbac/receiver-k8sobjects/01-assert.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
```

Comment:
This is fine now, but it would be better to do a real e2e tests and asserting the k8sobjects actually can collect data